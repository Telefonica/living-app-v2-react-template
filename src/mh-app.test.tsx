import renderer from 'react-test-renderer';
import MhApp from './mh-app';

test('<MhApp /> renders correctly', () => {
  const render = renderer.create(<MhApp />).toJSON();
  expect(render).toMatchSnapshot();
});
