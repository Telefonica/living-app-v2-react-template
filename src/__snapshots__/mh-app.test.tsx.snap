// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MhApp /> render: match snapshot 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
